vortex-file/src/io/file.rs

```diff
@@ -213,7 +213,9 @@ async fn evaluate<R: VortexReadAt>(
     }
 
     // Populate the cache
-    try_join_all(cache_futures).await?;
+    FuturesUnordered::from_iter(cache_futures)
```

Comment:
try_join_all polls all the futures, and we don't care about using the results. I don't understand what difference this makes? Am I misunderstanding?

Reply:
FuturesUnordered also polls all the futures but better. In all seriousness this is a minor change where try_join_all defaults to FuturesOrdered if there’s more than 30 futures and in this case futuresunordered would be marginally faster

Reply:
But why would it be faster? What's the actual difference?

Reply:
I went from the futures docs where they suggest directly constructing futuresordered/unordered if the number of them https://docs.rs/futures/latest/futures/future/fn.try_join_all.html#see-also. Afaik polling mechanism is different